MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -108,7 +114,7 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
      -  Returns: `MembraneRTC` client instance in connecting state
      */
     public static func connect(with options: ConnectOptions, delegate: MembraneRTCDelegate) -> MembraneRTC {
-        let client = MembraneRTC(eventTransport: options.transport, config: RTCConfiguration(), peerMetadata: options.config)
+        let client = MembraneRTC(eventTransport: options.transport, config: RTCConfiguration(), peerMetadata: options.config, encoder: options.encoder)
```

Comment:
We should update docs for `options` as they can contain encoder now

I am also wondering about `encoder` being a property of `track` so that we could create two tracks with different encoders each :thinking:

Reply:
> I am also wondering about encoder being a property of track so that we could create two tracks with different encoder each 🤔

I'm afraid it'd be a mess, because we're also choosing encodings in sdp negotiations. Also (correct me if I'm wrong) we can only set encoder in constructor of `RTCPeerConnectionFactory` so "globally" for every track at once. What happens when we renegotiate tracks after adding a track with different encoding? 
In case of simulcast encodings it was easier, because we could set it in peer connection sender params.

Is different encoder per track really a valid use case?

Reply:
> Is different encoder per track really a valid use case?

It was more a nice to have feature but it's perfectly okay to have the global setting at the moment. Let's add this feature when we find it to be useful